frontend/src/api.js

```diff
@@ -91,7 +98,15 @@ class Api {
       token
     });
 
-  logout = () => instance.post(`/user.logout`, {});
+  logout = () => {
+    if (isAuthProxyEnabled) {
+      const isUsingAuthproxy = store?.getState()?.get("login")?.toJS()?.isUsingAuthproxy;
+      if (isUsingAuthproxy && authProxySignoutUri) {
+        window.open(`${authProxySignoutUri}`, "_blank");
```

Comment:
@SamuelPull add detection of blocked popup and at maybe just display a message to the user? https://stackoverflow.com/a/2917/3198533

Reply:
@galethil Thank you, I was looking for a way how to handle this potential issue.
For testing purposes, I executed `window.open = function() { };`, because I could not block popups on localhost.